ruby/ql/test/library-tests/dataflow/summaries/Summaries.ql

```diff
@@ -117,23 +117,18 @@ private class SinkFromModel extends ModelInput::SinkModelCsv {
   override predicate row(string row) { row = "test;FooOrBar;Method[method].Argument[0];test-sink" }
 }
 
-class Conf extends TaintTracking::Configuration {
-  Conf() { this = "FlowSummaries" }
-
-  override predicate isSource(DataFlow::Node src) {
-    src.asExpr().getExpr().(StringLiteral).getConstantValue().isString("taint")
+class CustomValueSink extends DefaultValueFlowConf {
+  override predicate isSink(DataFlow::Node sink) {
+    super.isSink(sink)
+    or
+    sink = ModelOutput::getASinkNode("test-sink").getARhs()
   }
+}
 
+class CustomTaintSink extends DefaultTaintFlowConf {
   override predicate isSink(DataFlow::Node sink) {
-    exists(MethodCall mc |
-      mc.getMethodName() = "sink" and
-      mc.getAnArgument() = sink.asExpr().getExpr()
-    )
+    super.isSink(sink)
     or
     sink = ModelOutput::getASinkNode("test-sink").getARhs()
   }
 }
-
-from DataFlow::PathNode source, DataFlow::PathNode sink, Conf conf
-where conf.hasFlowPath(source, sink)
-select sink, source, sink, "$@", source, source.toString()
```

Comment:
I prefer to actually keep this, in addition to `import PathGraph`, like we do in other flow tests that use `InlineFlowTest`.